src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -59,6 +59,10 @@ using namespace conduit;
 // access verify logging helpers
 using namespace conduit::blueprint::utils;
 
+namespace conduit { namespace blueprint { namespace mesh {
+    bool verify_unidomain(const conduit::Node &n, conduit::Node &info);
```

Comment:
Minor style thingy:

instead of uni can we use verify_single_domain?
and verify_multi_domain?